docs/development-environment.md

```diff
@@ -259,34 +259,47 @@ To get started, there are several ways to run the unit tests, depending on how y
 
 ## PHP unit tests
 
-⚠️ This section is in need of update! Changes to how WordPress sets up its tests mean that a global version of phpunit is no longer provided.
-
 * ### Docker
 
 	To run the PHP unit tests for Jetpack if you're running Docker, you can run the following:
 
 	```sh
-	jetpack docker phpunit
+	jetpack docker phpunit jetpack
 	```
 
 	This will run unit tests for Jetpack. You can pass arguments to phpunit like so:
 
 	```sh
-	jetpack docker phpunit -- --filter=Protect
+	jetpack docker phpunit jetpack -- --filter=Protect
 	```
 
 	This command runs the tests as a multi site install
 
 	```sh
-	jetpack docker phpunit-jp-multisite -- --filter=Protect
+	jetpack docker phpunit jp-multisite -- --filter=Protect
 	```
 
-	To run tests for specific packages, you can run the tests locally, from within the package's directory:
+	To run tests for specific packages, you can run the tests locally. The most straightforward way is to use `jetpack test`, for example
```

Comment:
It seems a bit odd to provide detail in the Docker section rather than pointing people to a non-Docker section. `## JavaScript unit tests` seems to be a more relevant section (with a rename), but it doesn't currently mention the `js`/`php` options...